src/v/config/configuration.cc

```diff
@@ -512,6 +512,30 @@ configuration::configuration()
       "refreshed",
       {.visibility = visibility::tunable},
       2s)
+  , kafka_tcp_keepalive_idle_timeout_seconds(
+      *this,
+      "kafka_tcp_keepalive_timeout",
+      "TCP keepalive idle timeout in seconds for kafka connections. When "
+      "changed applies to new "
+      "connections only.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      120s)
+  , kafka_tcp_keepalive_probe_interval_seconds(
+      *this,
+      "kafka_tcp_keepalive_probe_interval_seconds",
```

Comment:
This is `tcp_keepalive_intvl` right?

So it's the retry interval when a probe failed, I think. We could make it clearer in the description, it sounds like it's the primary interval even when the KAs are being ACKed.

Reply:
Correct, will expand as mentioned above.